packages/frontend/src/routes/TransferWizardWrapper.js

```diff
@@ -38,22 +38,25 @@ export const TransferWizardWrapper = () => {
     return (
         <Container>
             <h1 >Migrating from Near Wallet</h1>
-            <p>As announced in our previous blog post, the Near wallet domain (<a target="blank" href="https://wallet.near.org">https://wallet.near.org</a>) will transition from a web wallet into a wallet hub, showcasing the multiple existing wallets maintained by our community. During and after the transition users will be encouraged to migrate their funds to other ecosystem wallets. To simplify the process, our team has created a Wallet Transfer Wizard that will guide you on every step. If you prefer to not use the wizard, you can move your funds manually using your 12-word recovery phrase or a hardware wallet.</p>
+            <p>As announced in an earlier (<a target="blank" href="https://medium.com/nearprotocol/near-opens-the-door-to-more-wallets-255eee58eb97">blog post</a>) on July 31, 2022, the NEAR wallet domain (https://wallet.near.org) will transition from a web wallet into a wallet hub, showcasing the multiple existing wallets maintained by the community. During and after the transition, users will be encouraged to migrate their accounts to other ecosystem wallets. To simplify the process, the Pagoda team has created a Wallet Transfer Wizard to guide users through every step. If users prefer to not use the Wizard, they may also migrate to a new wallet manually using your 12-word recovery phrase or a hardware wallet.</p>
```

Comment:
```
As announced in an earlier (blog post) on July 31...
```
There shouldn't be parens around `blog post` since it's directly referenced in the sentence.

```
showcasing the multiple existing wallets maintained by the community
```
Reads a little awkward to me - `showcasing the variety of wallets maintained by the community` seems more natural.

```
If users prefer to not use the Wizard, they may also...
```
🔪  - replace with `Users may also...` or `Users are still free to...`

```
manually using your 12-word recovery phrase or a hardware wallet.
```
Should be `manually using a 12-word recovery phrase or hardware wallet.`